mirrord-cli/src/main.rs

```diff
@@ -92,7 +94,15 @@ fn add_to_preload(path: &str) -> Result<()> {
     }
 }
 
+const CURRENT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
 fn exec(args: &ExecArgs) -> Result<()> {
+    let resp = reqwest::blocking::get("https://version.mirrord.dev/get-latest-version?source=2&currentVersion=0").unwrap().text().unwrap();
```

Comment:
this should fail silently/not crash

Reply:
ah tbh this should be on main after arg parse and not only on exec.
Also we should have a flag/environment variable that disables this check